recipes/piscem/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.0" %}
+{% set git_commit = "1ba1d8893b5135826bf13f7c5ceefcdc7e8f51b2" %}
+{% set sha256 = "218992c7ea649974b81fd459fd04f7bd98df4eab34af97ff227e7fa25b9e43f9" %}
+
+package:
+  name: piscem
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [osx]
+
+source:
+  git_url: https://github.com/COMBINE-lab/piscem
+  git_rev: "{{ git_commit }}"
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - rust >=1.60  
+    - cmake
+    - autoconf
+    - make
+    - zlib
```

Comment:
zlib, bzip2 jemalloc should only be in host, isn't it? Are they needed in build?

Reply:
Good question — I actually forgot the distinction here again, can you remind me or point me at the appropriate docs? Right now, my main focus is on trying to get the OSX build working ... which seems to be a huge pain.